Tools/CISupport/Shared/transfer-archive-to-s3

```diff
@@ -29,8 +29,8 @@ def main():
     parser = argparse.ArgumentParser(add_help=True)
 
     group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument('--revision', action="store", help='Revision number or patch_id for the built archive')
-    group.add_argument('--patch_id', action="store", help='patch_id of the patch')
+    group.add_argument('--revision', action="store", help='Revision number or change identifier for the built archive')
+    group.add_argument('--change-id', dest='change_id', action="store", help='patch id or hash of specified change')
```

Comment:
do you want to use `change-id` or `change_id`

Reply:
I think 'change-id' is more consistent with a command line option, even though the variable should clearly be 'change_id' in Python code. I did make the change deliberately.